dev/MRTCore/packaging/MrtCore.props

```diff
@@ -0,0 +1,37 @@
+<!--
+  Copyright (c) Microsoft Corporation. Licensed under the MIT License
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <_ReunionImageGlobs>**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif</_ReunionImageGlobs>
+  </PropertyGroup>
+
+  <!--
+    By default, add the .RESW files in the project to PRIResource and the image files in the project to Content.
+    That's needed for the strings in the .RESW files and the image files to be indexed during PRI generation.
+    The app dev can opt out a file by changing the Build Action file property to None in Visual Studio, or
+    by manually modifying the CSPROJ file.
+
+    The implementation here is the same as the .NET SDK's implementation for default file inclusion.
+    See https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview#default-includes-and-excludes.
+
+    We match the semantics of EnableDefaultCompileItems here: if EnableDefaultCompileItems is false, .CS files,
+    for example, are not added to the project, as opposed to added as None.
+
+    Yes, the properties used here, including EnableCoreMrtTooling, are defined in MSBuild files imported after
+    this file. This is correct though because ItemGroup definitions are evaluated in a later pass. See
+    https://docs.microsoft.com/en-us/visualstudio/msbuild/build-process-overview?view=vs-2019#evaluation-phase.
+  -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' and '$(EnableCoreMrtTooling)'=='true'">
```

Comment:
>'$(EnableCoreMrtTooling)'=='true'"> [](http://example.com/codeflow?start=74&length=35)

this is not true

Reply:
Item groups are evaluated in a later pass than properties. Please see https://docs.microsoft.com/en-us/visualstudio/msbuild/build-process-overview?view=vs-2019#evaluation-phase